### PR TITLE
Add data layer variable to GTM

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
- sends accountId to GTM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "watch": "gulp watch",
     "lint": "eslint .",
     "pretest": "npm-shrinkwrap-check -3 && eslint . && npm run compile && NODE_ENV=test npm run db-migrate",
-    "test": "NODE_ENV=test mocha && NODE_ENV=test npm run features",
+    "test-integration": "mocha",
```
Comment (nathancashmoredwp) [inline package.json]: ??

Comment (lukasz-kaniowski) [inline package.json]: just want to be able to run all integration tests using npm, i.e:
`npm run integration-test`

Comment (nathancashmoredwp) [inline package.json]: whats wrong with just
npm test

Comment (lukasz-kaniowski) [inline package.json]: you can still do `npm test` check line `12`

**Comment by nathancashmoredwp** [comment]
👍  ... will have a chat another time around how we can change the test strategy to be less implementation specific.

**Pull request merged**